File: Baekjoon_Online_Judge/NO_09498/9498.c
#include <stdio.h>
/*
 -------------------------------------- 
 | Created by happiness96             | 
 | Year 2020                          | 
 | Month 08                           | 
 | Day 31                             | 
 | 시험 성적                              | 
 -------------------------------------- 
*/

int main(){
    int score;

    scanf("%d", &score);

    if (90 <= score && score <= 100) printf("A");
    else if (80 <= score && score <= 89) printf("B");
    else if (70 <= score && score <= 79) printf("C");
    else if (60 <= score && score <= 69) printf("D");
    else printf("F");

    return 0;
}

File: Baekjoon_Online_Judge/NO_09498/9498.py
# -*- encoding: utf-8 -*-
import sys
r_input = sys.stdin.readline

# -------------------------------------- #
# | Created by happiness96             | #
# | Year 2020                          | #
# | Month 08                           | #
# | Day 31                             | #
# | 시험 성적                          | #
# -------------------------------------- #


if __name__ == "__main__":
    score = int(r_input())

    print('A' if 90 <= score <= 100 else 'B' if 80 <= score else 'C' if 70 <= score else 'D' if 60 <= score else 'F')
